payloads/external/Makefile.inc

```diff
@@ -137,7 +137,11 @@ payloads/external/depthcharge/depthcharge/build/depthcharge.elf depthcharge: $(D
 
 # Tianocore
 
-payloads/external/tianocore/tianocore/Build/UEFIPAYLOAD.fd tianocore: $(DOTCONFIG)
+ifeq ($(CONFIG_TIANOCORE_ENABLE_IPXE),y)
+IPXE_EFI := payloads/external/iPXE/ipxe/ipxe.rom
```

Comment:
If you add iPXE for EFI, then you will need changes to payload/external/iPXE/Makefile too